content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -78,7 +78,7 @@ The latency, throughput, and the proxies' CPU and memory consumption are measure
 ### CPU and memory
 
 Since the sidecar proxy performs additional work on the data path, it consumes CPU
-and memory. In Istio 1.21, a proxy consumes about 0.5 vCPU per 1000
+and memory. In Istio 1.22, a proxy consumes about 0.5 vCPU per 1000
```

Comment:
```suggestion
and memory. In Istio 1.22, a sidecar proxy consumes about 0.5 vCPU per 1000
```

Reply:
@Shuayb1 do we have stats here for ztunnel and waypoint?

Reply:
No, I didn't get that stat